### PR TITLE
Chart layout updates

### PR DESCRIPTION
**Pull request by rayhm6419**
Issues: https://github.com/1thing-org/1thing.org/issues/94

Try to match the figma design
image

2. Mobile design
image

3. Logo image size was fixed in the collapse toggle bar
image

### CODE REVIEW

File: src/components/home/Chart/IncidentChart.js [hunk 1]
```diff
@@ -111,13 +113,14 @@ const IncidentChart = ({ color, chart_data, state, isFirstLoadData }) => {
...
+        <LineChart
+          height={266}
+          width={955}
           data={chart_data}
           margin={{ top: 30, right: 30, bottom: 10, left: -30 }}
-          style={{ backgroundColor: "rgba(255, 255, 255, 0.1)" }}
+          style={{ backgroundColor: "#FBFBFB" }}
         >
-          <CartesianGrid strokeDasharray="3 3" />
+          {/* <CartesianGrid strokeDasharray="3 3" /> */}
```
Comment (jiayingxie) [inline src/components/home/Chart/IncidentChart.js:123]: NB: can we remove this commented code?

File: src/components/home/Chart/IncidentChart.js [hunk 2]
```diff
@@ -129,21 +132,22 @@ const IncidentChart = ({ color, chart_data, state, isFirstLoadData }) => {
...
-              fontWeight: 400,
+              fontWeight: 600,
               lineHeight: "16px",
               textAlign: "center",
               letterSpacing: "0.4px",
-              fill: "#F4F4F4",
+              fill: "var(--Dark-Grey, #535353)",
             }}
-            axisLine={{ stroke: "#FFFFFF", strokeWidth: 2 }}
+            // axisLine={{ stroke: "var(--Dark-Grey, #535353)", strokeWidth:  }}
```
Comment (jiayingxie) [inline src/components/home/Chart/IncidentChart.js:141]: Same here.

**Review by Chelseaguan0219 — APPROVED** [review]

File: src/components/home/Chart/ChartView.css [hunk 3]
```diff
@@ -18,32 +18,46 @@
...
+  display: flex;
+  width: 201px;
+  height: 61px;
+  padding: 10px;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  border-radius: var(--Redius, 2px);
+  border: 1px solid var(--Primary-color, #ffc530) !important;
+  background: var(--Primary-color, #ffc530) !important;
```
Comment (jiayingxie) [inline src/components/home/Chart/ChartView.css:60]: NB: is it possible not using `!important`?

**Review by jiayingxie — APPROVED** [review]
Left some comments, not blocking.

**Pull request merged**